appstudio-utils/util-scripts/lib/fetch/git.sh

```diff
@@ -16,6 +16,12 @@ git-fetch-repo() {
 }
 
 git-fetch-policies() {
+  local repository_and_ref
+  if repository_and_ref=$(kubectl get enterprisecontractpolicies.appstudio.redhat.com ec-policy -o jsonpath='{.spec.sources[0].git.repository}#{.spec.sources[0].git.revision}'); then
```

Comment:
Shouldn't `ec-policy` not be hard coded?

Reply:
Doh